ResearchKit/Common/ORKSurveyAnswerCell.h

```diff
@@ -43,6 +43,7 @@ NS_ASSUME_NONNULL_BEGIN
 @required
 - (void)answerCell:(ORKSurveyAnswerCell *)cell answerDidChangeTo:(id)answer dueUserAction:(BOOL)dueUserAction;
 - (void)answerCell:(ORKSurveyAnswerCell *)cell invalidInputAlertWithMessage:(NSString *)input;
+- (void)answerCell:(ORKSurveyAnswerCell *)cell errorAlertWithTitle:(NSString *)title message:(NSString *)message;
```

Comment:
Refactor to match with `ORKFormItemCell`. Should say `invalidInputAlertWithTitle: message:`.